paddle/fluid/pybind/eager_method.cc

```diff
@@ -716,6 +716,15 @@ static PyObject* set_grad_type(TensorObject* self, PyObject* args,
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+static PyObject* tensor__inplace_version(TensorObject* self, PyObject* args,
```

Comment:
single underscore "_" in function name?

Reply:
its ok if this method indicate `_inplace_version` api in python